elm.json

```diff
@@ -0,0 +1,29 @@
+{
+    "type": "package",
+    "name": "PaackEng/elm-svg-string",
+    "summary": "Serializes a SVG node into a string",
+    "license": "BSD-3-Clause",
+    "version": "0.0.1",
```

Comment:
I think by default the package version should start at `1.0.0`, when you try to publish you will get an error. That being said I think it's better to set to `1.0.0`